esphome/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_ESPHOME_IP
+      APP_PORT: 6052
+
+  server:
+    image: esphome/esphome:2022.6.2@sha256:1fac6334b85e76c7fd936036891b7a7ee761ee73fd119662b2cb564c3faf818f
+    volumes:
+      - ${APP_DATA_DIR}/data:/config
+      - /etc/localtime:/etc/localtime:ro
```

Comment:
I don't like mounting this in from the host. It looks like this is only used for logging: https://github.com/esphome/issues/issues/398

There is a semi-interesting point, that this approach will also constantly make syscalls.

Perhaps as an alternative (as suggested), we set the `TZ` env. var. to `UTC` ?

Reply:
Alright, I'll remove it

Also, `UTC` should already be the default